plugins/GSdx/GSSettingsDlg.cpp

```diff
@@ -767,6 +767,11 @@ bool GSHacksDlg::OnMessage(UINT message, WPARAM wParam, LPARAM lParam)
 
 			EndDialog(m_hWnd, id);
 		} break;
+
+		case IDCANCEL:
+		{
+			EndDialog(m_hWnd, IDCANCEL);
+		} break;
```

Comment:
Thinking again, even though the ``break`` is redundant, it's still a good coding practice to always have it on case statements. And I'm afraid Coverity will complain if we remove the break so let's have it :+1: